package.json

```diff
@@ -95,6 +95,7 @@
   "dependencies": {
     "@changesets/cli": "^2.16.0",
     "@codechecks/client": "^0.1.11",
-    "@ethersproject/abstract-provider": "^5.7.0"
+    "@ethersproject/abstract-provider": "^5.7.0",
+    "changeset": "^0.2.6"
```

Comment:
Why is [this](https://www.npmjs.com/package/changeset) needed?

Reply:
hmmm, I think you must have run `yarn add changeset` to cause that diff. All good though, thanks.

Reply:
You are right, my bad I did run `yarn add changeset`

Reply:
https://github.com/ethereum-optimism/optimism/pull/4003#issuecomment-1315682790

Your changes look good, but we accidentally introduced a linter error to `develop` which is causing this branch to fail in CI. You should be able to fix this by rebasing your branch on develop. I believe these commands should achieve that:

```
git pull --rebase https://github.com/ethereum-optimism/optimism develop
git push --force
```